config/initializers/money.rb

```diff
@@ -58,6 +57,8 @@
   config.add_rate "USD", "US8", 1
```

Comment:
I understand that you will need to add rates for EUR and all other currencies supported then. That would help as you can use the helper to translate to any currency in the interface.